assumptions.md

```diff
@@ -2,15 +2,13 @@
 There are some assumptions about the data that are inherent to FHD and its mathematical basis. Documented below are these assumptions and their consequences. <br />
 
 ## Time-based <br />
-
-**Length of observations** <br />
-PSFs are calculated only once per snapshot (ObsID). Sources will move through the beam over the course of a given snapshot, but the assumption is that the sky is stationary. This leads to small discontinuities visible in waterfall plots. Choose snapshots shorter than about 2 minutes to minimize the discontinuity between adjacent snapshots. Also, phase to the zenith at the center of the snapshot.
+PSFs are calculated only once per observation. Sources will move through the beam over the course of a given snapshot, but the assumption is that the sky is stationary. This leads to small discontinuities visible in waterfall plots. Choose snapshots shorter than about 2 minutes to minimize the discontinuity between adjacent snapshots. Also, phase to the zenith at the center of the snapshot.
```

Comment:
Sorry, just looking through this and I think this isn't quite right. I think the primary beam is only calculated once per observation, but the PSF (which depends on the baseline uvws is calculated separately for each integration inside the observation. So I think the text is correct except that the first word should be Primary beam rather than PSF.